pkgs/games/vvvvvv/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchurl
+, cmake
+, makeWrapper
+, copyDesktopItems
+, makeDesktopItem
+, pkgconfig
```

Comment:
```suggestion
, pkg-config
```

Reply:
`pkg-config` isn't actually needed at all, but I messed up with Git and accidentally put it back without noticing (sorry for the mistake).